sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -2647,7 +2647,11 @@ object functions {
   }
 
   /**
-   * Gets current Unix timestamp in seconds.
+   * Returns the current Unix timestamp (in seconds).
+   *
+   * NOTE: All calls of `unix_timestamp` within the same query return the same value
```

Comment:
Should we use `@note` and fix the corresponding contents in Python and R if applicable?